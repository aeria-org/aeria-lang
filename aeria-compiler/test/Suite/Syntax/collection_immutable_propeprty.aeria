collection CollectionImmutableListTest {
  immutable {
    prop1
    prop2
  }
}
