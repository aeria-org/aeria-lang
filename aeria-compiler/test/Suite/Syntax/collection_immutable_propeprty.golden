{
  "kind": "Program",
  "collections": [
    {
      "name": {
        "value": "CollectionImmutableListTest",
        "kind": "CollectionName"
      },
      "kind": "Collection",
      "immutable": {
        "kind": "CollectionImmutableList",
        "immutables": [
          {
            "value": "prop1",
            "kind": "PropertyName"
          },
          {
            "value": "prop2",
            "kind": "PropertyName"
          }
        ]
      }
    }
  ]
}
